crates/roc_std/src/roc_list.rs

```diff
@@ -147,6 +147,92 @@ impl<T> RocList<T>
 where
     T: Clone,
 {
+    pub fn from_slice(slice: &[T]) -> Self {
+        let mut list = Self::empty();
+        list.extend_from_slice(slice);
+        list
+    }
+
+    pub fn extend_from_slice(&mut self, slice: &[T]) {
+        // TODO: Can we do better for ZSTs? Alignment might be a problem.
+        if slice.is_empty() {
+            return;
+        }
+
+        let new_len = self.len() + slice.len();
+        let non_null_elements = if let Some((elements, storage)) = self.elements_and_storage() {
+            // Decrement the list's refence count.
+            let mut copy = storage.get();
+            let is_unique = copy.decrease();
```

Comment:
@folkertdev So, removing the decrement caused a CI failure (including after I tried re-running on the same commit, in case it was a flake), and reverting it fixed it. 🤨

I don't understand why that would be, though. Any ideas?

Reply:
actually, yes. 

in roc, this function would take its argument as owned. That means the argument must be consumed, which is what that decrement does.